packages/client/src/components/SafeOverview.js

```diff
@@ -9,7 +9,7 @@ function SafeOverview({ allBalance }) {
     "p-5 mr-6",
   ];
 
-  const tokensWithPositiveBalance = allBalance
+  let tokensWithPositiveBalance = allBalance
```

Comment:
Was this erroring out?

Reply:
yes this was unrelated error trying to re-assign a const

Reply:
Thanks! I'll look into that